x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.tsx

```diff
@@ -120,8 +97,8 @@ export const Result: React.FC<Props> = ({
           resultMeta={resultMeta}
           showScore={!!showScore}
           isMetaEngine={isMetaEngine}
-          shouldLinkToDetailPage={shouldLinkToDetailPage}
-          actions={<ResultActions />}
+          documentLink={documentLink}
+          actions={actions}
```

Comment:
I could have left this as `actions={<ResultActions actions={actions} />}` but moving `<ResultActions>` to ResultHeader made it easier to unit test without diving, and the conditional logic pattern there matched the rest of the conditional header items.